src/I18n/I18n.php

```diff
@@ -201,6 +201,18 @@ public static function config($name, callable $loader)
     }
 
     /**
+     * Registers a custom fallback translator loader
+     *
+     * @param callable $loader A callable object that should return a Package
+     * instance to be used for assembling a new translator.
+     * @return void
+     */
+    public static function fallbackConfig(callable $loader)
```

Comment:
Fair point. Would it be reasonable to have:

```php
I18n::config('_fallback', $customLoader)
```

Then we don't need any new methods, and fallback isn't set in a different way than other loaders, it just has a special name.